File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    odd_concat = ''
    even_concat = ''
    for i in num_list:
        if i % 2 == 0:
            even_concat = even_concat + str(i)
        else:
            odd_concat = odd_concat + str(i)
    return int(odd_concat) + int(even_concat)

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        String even = "";
        String odd = "";
        for (int idx = 0; idx < num_list.length; idx++) {
            if (num_list[idx] % 2 == 0) {
                even += String.valueOf(num_list[idx]);
            } else {odd += String.valueOf(num_list[idx]);}
        }
        answer = Integer.parseInt(even) + Integer.parseInt(odd);
        return answer;
    }
}